src/UI/Internal/Text.elm

```diff
@@ -1,4 +1,4 @@
-module UI.Internal.Text exposing (..)
+module UI.Internal.Text exposing (Options, Properties, Text(..), TextColor(..), TextSize(..), attributes, defaultColor, defaultText, deskAttributes, fontColor, mobileAttributes)
```

Comment:
I oppose that we should care about what an internal module exposes or not, that's the whole concept of having internal functions. They're here to not be used, but if needed there's no limit to what we can internally do.